### DISCUSSION TITLE
RHEL CI/Repo

### DISCUSSION THREAD

**Discussion by mskarbek** [openzfs/zfs#11561] [Ideas]
Now, with the new, extended RHEL Developer program do you think that it may be reasonable to look at extending at least CI with RHEL-based worker?

**Comment by behlendorf** [comment]
Yes, I think this is something we'll want to look in to.

**Reply by mskarbek** [reply]
https://www.redhat.com/en/blog/extending-no-cost-red-hat-enterprise-linux-open-source-organizations
